### ISSUE TITLE
Any plans to release the dataset?

### ISSUE THREAD

**Issue by cmyopi** [vLAR-group/PhysInOne#1]
Hi there,

Thank you for sharing the PhysInOne project! I've been reading through your work and I'm really impressed by what you've achieved here. It's a fantastic piece of work.

I was wondering if you have any plans to open-source the dataset in the near future? I would love to experiment with it and use it as a benchmark for my own research.

**Comment by Jinxi-Li** [comment]
Hi,

Super thanks for your interest in our work! Since this is a huge dataset, we are still organizing it. And we do have plan to release it step-by-step. 

STEP 1. we have released part of validation set in [huggingface](https://huggingface.co/datasets/vLAR/PhysInOne), you can download and play with it at current time.
STEP 2. we will release all the validation set and test set hopefully within this month.
STEP 3. we will release the related codes in this github repo. 
STEP 4. we will release all the assets and scenes and rendered results of training setting in the future.

Siyuan @MT-Shark  will make an in-detail plan and update it in the README. 

Thanks for your interest again! Keep tuned!

Best,
Jinxi
